modules/tinymce/src/themes/silver/main/ts/ui/toolbar/Integration.ts

```diff
@@ -199,7 +199,7 @@ const identifyButtons = (editor: Editor, toolbarConfig: RenderToolbarConfig, bac
         lookupButton(editor, toolbarConfig.buttons, toolbarItem, toolbarConfig.allowToolbarGroups, backstage, prefixes).toArray();
     });
     return {
-      title: Optional.from(editor.translate(group.name)),
+      title: Optional.none(),
```

Comment:
This needs to be rendered as something maybe a `aria-label` since with this change the `name` has no use and the screen reader will no longer read out the toolbar groups at least not VoiceOver. Before this change it would say `Entering formatting` and `Leaving history` etc when you tab into groups. With this change it says nothing. I tested adding it as a `aria-label` and at least VoiceOver will speak that. So I suggest we make it a `aria-label` then get QA to see if that works on Jaws, other browsers etc.

- [ ] Change `name` to `aria-label`

Reply:
Alright.